src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -328,12 +328,14 @@ public IObservable<Unit> SwitchToBranch(ILocalRepositoryModel repository, IPullR
             });
         }
 
-        public Encoding GetEncoding(string path)
+        public Encoding GetEncoding(ILocalRepositoryModel repository, string relativePath)
         {
-            if (File.Exists(path))
+            var fullPath = Path.Combine(repository.LocalPath, relativePath);
+
+            if (File.Exists(fullPath))
```

Comment:
Also if a file has been added in a PR, but that PR is not checked out then it won't exist on disk.